### PR TITLE
Added SSRF Finding

### PR DESCRIPTION
**Pull request by sethsec**

### CODE REVIEW

File: Web/Server-Side Request Forgery.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+/*
+Title: Server-Side Request Forgery (SSRF)
+Description: Search engine meta data about the finding
```
Comment (mubix) [inline Web/Server-Side Request Forgery.md]: can you add some basic keywords or meta data for search engines to pick up

File: Web/Server-Side Request Forgery.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+- [OWASP Top 10 2013-A1-Injection](https://www.owasp.org/index.php/Top_10_2013-A1-Injection)
+- [CWE-918: Server-Side Request Forgery (SSRF)](http://cwe.mitre.org/data/definitions/918.html)
+- [CWE-441: Unintended Proxy or Intermediary ('Confused Deputy')](http://cwe.mitre.org/data/definitions/441.html)
+- [EC2 Instance Metadata Service Documentation](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html)
+- [OpenStack Metadata Service Documentation](http://docs.openstack.org/admin-guide-cloud/content/section_metadata-service.html)
+- [Compromising an unreachable Solr server with CVE-2013-6397](http://www.agarri.fr/kom/archives/2013/11/27/compromising_an_unreachable_solr_server_with_cve-2013-6397/index.html)
+- [Bringing a Machete to the Amazon](https://www.youtube.com/watch?v=JTOWxi17k-w)
+- [Prezi Got pwned - A tale of responsible disclosure](http://engineering.prezi.com/blog/2014/03/24/prezi-got-pwned-a-tale-of-responsible-disclosure/)
+
+## Exploitation
```
Comment (mubix) [inline Web/Server-Side Request Forgery.md:49]: I'd like a bit more demo in this section (pictures etc) but I'll accept as is with the meta data changed above, and the request that you come back and make a pull request with some pictures once I make a path to do so.

**Comment by sethsec** [comment]
Added metadata as requested.  Yea. Definitely willing to update with some pictures once you decide on how you want to handle that.

**Comment by mubix** [comment]
please put any images in the "imgs/" directory and reference them using `` format.

**Comment by sethsec** [comment]
Should be good to go now.  When I used `` as requested, it was looking in `...cfdb/web/imgs/web/ssrf1.png`.  

I'm still a git n00b, so the only way I could get it to work was putting an `imgs` directory inside of `web`, which is what you can see now - ex: ``.   

Let me know if what you see is not what you envisioned.

**Pull request merged**